### PR TITLE
2023 redesign assets

### PR DESCRIPTION
**Pull request by Lordfirespeed**
This should be all the necessary assets for the proposed website design (so far). 

There's some interesting stuff going on with SVG `<use>` tags, especially for the Megateam icons, that allows assets to be nested and re-used from other files. (Browser compatability is all [good](https://caniuse.com/mdn-svg_elements_use) 😉)

### CODE REVIEW

File: public/assets/graphics/stars/artifact-stars-upper.svg
Comment (tameTNT) [inline public/assets/graphics/stars/artifact-stars-upper.svg:1]: Is there a reason the two stars artifact svgs are red tinted btw? Would be easier if we do a quick edit to white for them both? I can't determine whether it's to do with the blend mode maybe?

File: public/assets/graphics/artifact-planet.svg
Comment (tameTNT) [inline public/assets/graphics/artifact-planet.svg:1]: Ignore this - fixed failed. Working on it 🙃

**Review by tameTNT — APPROVED** [review]

**Comment by tameTNT** [comment]
File structure as well as files themselves look good! 👍
Future assets can be dropped in their respective feature branches directly instead of needing a full PR (since there'll be far less of them) 🫡

File: public/assets/graphics/stars/artifact-stars-upper.svg
Comment (Lordfirespeed) [reply public/assets/graphics/stars/artifact-stars-upper.svg:1]: They are red-tinted in Figma, too. Colour theory is not my area of expertise 😅 .
Remember they will be on a blue-black background and the red 'tint' has transparency. Somehow that leads to this:
image

Comment (Lordfirespeed) [reply public/assets/graphics/stars/artifact-stars-upper.svg:1]: I think you're right - it's the blend mode `color dodge` (CSS: `mix-blend-mode: color-dodge;`)

Comment (Lordfirespeed) [reply public/assets/graphics/stars/artifact-stars-upper.svg:1]: Okay, I get what's going on - the styling applied is *localised to the SVG*. that means that although the svg contains styling for `mix-blend-mode` the svg's elements won't blend with the HTML background as they are supposed to. 
Instead, we should style the element referencing the SVG's `mix-blend-mode`.

**Pull request merged**